UFS Utils fails to build on C6 following maintenance

Following yesterday's maintenance (6/17/2025), the build fails to build on Gaea C6 with the error message
```
-- Check for working C compiler: /opt/cray/pe/craype/2.7.30/bin/cc - broken
CMake Error at /autofs/ncrc-svm1_proj/epic/spack-stack/c6/spack-stack-1.6.0/envs/unified-env/install/intel/2023.2.0/cmake-3.23.1-ixiyqlk/share/cmake-3.23/Modules/CMakeTestCCompiler.cmake:69 (message):
  The C compiler

    "/opt/cray/pe/craype/2.7.30/bin/cc"

  is not able to compile a simple test program.

  It fails with the following output:

    Change Dir: /gpfs/f6/drsa-hurr1/scratch/David.Huber/UFS_UTILS/develop/build/CMakeFiles/CMakeTmp

    Run Build Command(s):/usr/bin/gmake -f Makefile cmTC_9f759/fast && /usr/bin/gmake  -f CMakeFiles/cmTC_9f759.dir/build.make CMakeFiles/cmTC_9f759.dir/build
    gmake[1]: Entering directory '/gpfs/f6/drsa-hurr1/scratch/David.Huber/UFS_UTILS/develop/build/CMakeFiles/CMakeTmp'
    Building C object CMakeFiles/cmTC_9f759.dir/testCCompiler.c.o
    /opt/cray/pe/craype/2.7.30/bin/cc   -gcc-name=/usr/bin/gcc-12  -o CMakeFiles/cmTC_9f759.dir/testCCompiler.c.o -c /gpfs/f6/drsa-hurr1/scratch/David.Huber/UFS_UTILS/develop/build/CMakeFiles/CMakeTmp/testCCompiler.c
    Error:  Unable to find cray-libsci/25.03.0 libraries compatible with intel-classic/2023.2.0 targeting 'x86-genoa'.
```

Other systems, like the GSI, are able to compile as of today.  Investigation reveals that the following `unload` may need to be added to `build.gaeac6.intel.lua`:
```diff
diff --git a/modulefiles/GDAS/gaeac6.intel.lua b/modulefiles/GDAS/gaeac6.intel.lua
index 9069da3..55a6afa 100644
--- a/modulefiles/GDAS/gaeac6.intel.lua
+++ b/modulefiles/GDAS/gaeac6.intel.lua
@@ -75,6 +75,7 @@ load("py-scipy/1.11.3")
 load("py-xarray/2023.7.0")
 load("py-f90nml/1.4.3")
 load("py-pip/23.1.2")
+unload("cray-libsci")
```

Doing so allowed UFS Utils to build.

--- a/modulefiles/build.gaeac6.intel.lua
+++ b/modulefiles/build.gaeac6.intel.lua
@@ -62,5 +62,7 @@ load(pathJoin("esmf", esmf_ver))
 nco_ver=os.getenv("nco_ver") or "5.0.6"
 load(pathJoin("nco", nco_ver))
 
+unload("cray-libsci")
+
 whatis("Description: UFS_UTILS build environment")
 

